packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -966,20 +982,50 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 
 	/**
 	 * {@inheritdoc IIntervalCollection.add}
+	 * @deprecated - call IntervalCollection.add without specifying an intervalType
 	 */
 	public add(
 		start: number,
 		end: number,
 		intervalType: IntervalType,
 		props?: PropertySet,
+		stickiness?: IntervalStickiness,
+	): TInterval;
+
+	public add(
+		start: number,
+		end: number,
+		props?: PropertySet,
+		stickiness?: IntervalStickiness,
+	): TInterval;
+
+	public add(
+		start: number,
+		end: number,
+		intervalType: IntervalType | PropertySet | undefined,
+		props?: PropertySet | IntervalStickiness,
 		stickiness: IntervalStickiness = IntervalStickiness.END,
-	): TInterval {
+	) {
+		let type: IntervalType;
+		let properties: PropertySet | undefined;
+		let sticky = stickiness;
+		if (intervalType !== undefined && typeof intervalType === "number") {
+			type = intervalType;
+			if (props === undefined || typeof props === "object") {
+				properties = props;
+			}
+		} else {
+			type = IntervalType.SlideOnRemove;
+			properties = intervalType;
+			if (typeof props === "number") {
+				sticky = props;
+			}
+		}
+
 		if (!this.localCollection) {
 			throw new LoggingError("attach must be called prior to adding intervals");
 		}
-		if (intervalType & IntervalType.Transient) {
-			throw new LoggingError("Can not add transient intervals");
-		}
+
```

Comment:
let's leave this throw in for now (but using `type` instead). We can remove it when we clean up the old signature. Right now people could still pass in an explicit intervalType if they use the deprecated API.